### PR TITLE
Ehnc radial progress

### PR DESCRIPTION
**Pull request by msecret**
Not exactly to comps but not bad either. Can adjust the width of things. Will have trouble making it responsive because uses absolute measurements.

screenshot from 2014-12-09 23 06 42

### CODE REVIEW

File: peacecorps/peacecorps/static/peacecorps/css/src/components/_radial_progress.scss [hunk 1]
```diff
@@ -5,3 +5,47 @@
...
+        transparent), linear-gradient($nextdeg, $barColor 50%, $backColor 50%,
+        $backColor);
+    } @else {
+      $nextdeg: -90deg + ($increment * ( $i - $half ));
+      background-image: linear-gradient($nextdeg, $barColor 50%, transparent
+        50%, transparent), linear-gradient(270deg, $barColor 50%, $backColor 50%,
+        $backColor);
+    }
+  }
+}
```
Comment (cmc333333) [inline peacecorps/peacecorps/static/peacecorps/css/src/components/_radial_progress.scss]: This loop in the style sheet feels a little hacky. Why not have this inline in the template? That'd allow for arbitrary precision.

**Comment by cmc333333** [comment]
Please merge upstream -- github's not doing a good job of showing what's changed

File: peacecorps/peacecorps/static/peacecorps/css/src/components/_radial_progress.scss (hunk 1 above)
Comment (msecret) [inline peacecorps/peacecorps/static/peacecorps/css/src/components/_radial_progress.scss]: I wanted to include this in the template too. I took some of this code from elsewhere, so haven't really hacked on it yet.

Comment (cmc333333) [inline peacecorps/peacecorps/static/peacecorps/css/src/components/_radial_progress.scss]: Gotcha. I'll make a ticket for that and we can merge this once upstream is pulled in.

**Comment by cmc333333** [comment]
Bump - was this merged in with another pull request? If not, I'd bet the conflicts are mostly with the compiled code.

File: peacecorps/peacecorps/templates/donations/includes/project-top.jinja [hunk 2]
```diff
@@ -32,6 +32,25 @@
...
+      <section class="u-align_c inline_grid--15">
+        {% set percent_completed = project.account.percent_funded() %}
+        <div class="radialProgress u-h_center--block"
+          {% if percent_completed < 50 %}
+            {% set deg = 90 + (3.6 * percent_completed) %}
+          style="background-image: linear-gradient(90deg, #F2EFE8 50%, transparent 50%, transparent), linear-gradient({{ deg }}deg, #A12122 50%, #F2EFE8 50%, #F2EFE8);"
+          {% else %}
+            {% set deg = -90 + (3.6 * (percent_completed - 50)) %}
+          style="background-image: linear-gradient({{ deg }}deg, #A12122 50%, transparent 50%, transparent), linear-gradient(270deg, #A12122 50%, #F2EFE8 50%, #F2EFE8);"
+          {% endif %}
```
Comment (cmc333333) [inline peacecorps/peacecorps/templates/donations/includes/project-top.jinja:44]: Nice

**Pull request merged**